<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskDialog</class>
 <widget class="QWidget" name="TaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_pLabelTaskName">
       <property name="text">
        <string>Task Name</string>
       </property>
       <property name="buddy">
        <cstring>m_pLineEditTaskName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_pLineEditTaskName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Begin</string>
       </property>
       <property name="buddy">
        <cstring>m_pTimeEditBegin</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="m_pTimeEditBegin">
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Duration:</string>
       </property>
       <property name="buddy">
        <cstring>m_pTimeEditDuration</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="m_pTimeEditDuration">
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Alarm:</string>
       </property>
       <property name="buddy">
        <cstring>m_pComboBoxAlarmBefore</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_pComboBoxAlarmBefore">
       <item>
        <property name="text">
         <string>0 Minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 Minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 Minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15 Minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30 Minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 Hour</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 Hours</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 Hours</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_pLabelDescription">
     <property name="text">
      <string>Task Desription</string>
     </property>
     <property name="buddy">
      <cstring>m_pTextEditTaskDescription</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="m_pTextEditTaskDescription">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QWidget" name="m_pWidgetBrowseButtons" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="m_pButtonFirstTask">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rsc/resources.qrc">
            <normaloff>:/img/go-top.png</normaloff>:/img/go-top.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_pButtonPreviousTask">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rsc/resources.qrc">
            <normaloff>:/img/go-up.png</normaloff>:/img/go-up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_pButtonNextTask">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rsc/resources.qrc">
            <normaloff>:/img/go-down.png</normaloff>:/img/go-down.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_pButtonLastTask">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rsc/resources.qrc">
            <normaloff>:/img/go-bottom.png</normaloff>:/img/go-bottom.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="m_pOkAbbortButtonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pLineEditTaskName</tabstop>
  <tabstop>m_pTimeEditBegin</tabstop>
  <tabstop>m_pTimeEditDuration</tabstop>
  <tabstop>m_pComboBoxAlarmBefore</tabstop>
  <tabstop>m_pTextEditTaskDescription</tabstop>
  <tabstop>m_pOkAbbortButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../rsc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
